common/rdr/HexInStream.cxx

```diff
@@ -21,13 +21,16 @@
 #include <config.h>
 #endif
 
+#include <algorithm>
 #include <rdr/HexInStream.h>
 #include <rdr/Exception.h>
 #include <rfb/util.h>
 
-using namespace rdr;
+#if defined(_MSC_VER)
+#undef min
+#endif
```

Comment:
This is a surprising new addition. Are you saying that std::min cannot be safely used in Visual Studio?

Reply:
In fact, the mix macro is defined in the minwindef.h header file, while Windows.h references minwindef.h. The mix macro is defined as follows:
```
#ifndef NOMINMAX
 
#ifndef max
#define max(a,b)            (((a) > (b)) ? (a) : (b))
#endif
 
#ifndef min
#define min(a,b)            (((a) < (b)) ? (a) : (b))
#endif
 
#endif  /* NOMINMAX */
```

Maybe it's better to define  NOMINMAX